index.bs

```diff
@@ -2099,6 +2103,45 @@ Security, Privacy, and Comfort Considerations {#security}
 
 The WebXR Device API provides powerful new features which bring with them several unique privacy, security, and comfort risks that user agents must take steps to mitigate.
 
+Sensitive Information {#sensitive-information-header}
+---------------------
+
+In the context of XR, <dfn>sensitive information</dfn> includes, but is not limited to, user configurable data such as interpupillary distance (IPD) and sensor-based data such as {{XRPose}}s. All {{XRSessionMode/"immersive-vr"}} and {{XRSessionMode/"immersive-ar"}} sessions will expose some amount of sensitive data, due to the user's pose being necessary to render anything. However, in some cases, the same sensitive information will also be exposed via {{XRSessionMode/"inline"}} sessions.
+
+Trustworthy Documents and Origins {#trustworthy-documents}
```

Comment:
TIL that the term "potentially trustworthy" has [an established meaning](https://www.w3.org/TR/secure-contexts/#is-origin-trustworthy) within secure contexts. Any concerns about using similar terminology in WebXR? Sorry @NellWaliczek that I didn't catch this before.